root/user/UserProfile.js

```diff
@@ -875,6 +875,24 @@ component UserProfile(
   if (adminViewing && isUntrusted(user)) {
     restrictions.push(l_admin('Untrusted'));
   }
+  const searchEditNotesURL = $c.user == null
```

Comment:
I used `==` rather than `===` here to match the `$c.user != null` checks on L870 and L871. (Can this be undefined, or is it guaranteed to be set to null?)